res/skins/Deere/style.qss

```diff
@@ -296,182 +376,101 @@ QTreeView::branch:open:has-children:has-siblings {
   height: 6px;
 }
 
-/* QSplitter between LibrarySidebar and CoverArt */
```

Comment:
If you enable ``View > Show Cover Art``, the horizontal splitter handle does not display the horizontal image, it uses the OS-specific default style instead.